Commandant/HelpCommand.swift

```diff
@@ -63,6 +63,10 @@ public struct HelpCommand: CommandType {
 
 private struct HelpOptions: OptionsType {
 	let verb: String?
+	
+	init(verb: String?) {
+		self.verb = verb
+	}
```

Comment:
Yeah, the compiler complained about the (nonexistent) initializer not initializing all properties otherwise. :unamused: